### ISSUE TITLE
Account creation -  Email changes

### ISSUE THREAD

**Issue by Sogomn** [ludumdare/ludumdare#407]
Sorry for creating a new issue for this but I haven't found a better place for this. I struggle creating an account for the new LD website. I've changed my email recently but it still has linked my name to my old address which I have no longer access to. Is there any way to re-create that account nevertheless? The user name is "*Sogomn*".

**Comment by zwrawr** [comment]
> Don’t have access to your old e-mail account? That’s okay! Create a temporary account with an e-mail you do have access to. Next year, when we start migrating the data over, we’ll have a process for recovering old names and accounts.
*[Third post on ldjam.com](https://ldjam.com/alpha-testing-update-1)*

**Comment by Sogomn** [comment]
Oh I'm sorry, I missed that post.

**Comment by povrazor** [comment]
I've re-opened the issue, since it is an issue.

Recovering an old account/data requires that you have access to the email set on `ludumdare.com`.

**NEW**: This doesn't work yet, apparently you **CAN** Edi your e-mail address on the old site. If you are able to log in to your account (despite not having access to the email), do it, and change the email. Sometime next year I'm going to redump the name list, and this will speed things up for you.

**Comment by zwrawr** [comment]
Related to #385

**Comment by Sogomn** [comment]
Yes I did that before but apparently the new site think I'm still on my old address.

**Comment by povrazor** [comment]
Yeah. :cry:

For the time being, if you haven't already, create an account using a temporary name. You'll be able to change it later.

**Comment by Sogomn** [comment]
It's not a big deal. As long as my desired name is reserverd... 😉
